spec/models/tweet_spec.rb

```diff
@@ -21,5 +21,40 @@
 require 'rails_helper'
 
 RSpec.describe Tweet, type: :model do
-  # pending "add some examples to (or delete) #{__FILE__}"
+  let(:tweet) { FactoryBot.build(:tweet, text: 'test tweet') }
+
+  # 正常系
+  describe 'type valid' do
+    it 'ツイートにテキストがあり、ユーザーと紐づいている' do
+      expect(tweet).to be_valid
+    end
+
+    it 'ツイートをいいねできる' do
+      user = FactoryBot.create(:user, email: 'user@example.com', password: 'password')
+      tweet.save
+      tweet.liked_by(user)
+      expect(Like.where(user: user)).to_not eq nil
+    end
+  end
+
+  # 異常系
+  describe 'type invalid' do
+    it 'ツイートが空' do
+      tweet.update(text: nil)
+      tweet.save
+      expect(tweet.errors[:text]).to include('を入力してください')
+    end
+
+    it 'ユーザーが紐づいていいない' do
+      tweet = FactoryBot.build(:tweet, user: nil)
+      tweet.save
```

Comment:
saveはdbが絡むのでたくさんテストすると地味に時間がかかります。
valid?を使いましょう。
```suggestion
      tweet.valid?
```